src/Frontend/Page/Topic.hs

```diff
@@ -189,14 +201,16 @@ instance FormPage CreateTopic where
 
     redirectOf (CreateTopic _ _ _) = U.listTopicIdeas
 
-    makeForm (CreateTopic space ideas timestamp) =
+    makeForm CreateTopic{ _createTopicIdeaSpace
+                        , _createTopicIdeas
+                        , _createTopicRefPhaseEnd } =
```

Comment:
Ok, I agree that this is an improvement.  Will try to start using that myself.  (-: